admin/proc-rhdh-deploy-eks-using-operator.adoc

```diff
@@ -0,0 +1,524 @@
+              port:
+                name: http-backend
+
+----
+
+[NOTE]
+====
+Configuring the HTTPS listener with Application Load Balancer (ALB) requires a certificate for your custom domain. In case, the custom domain or certificate is not available, then you can set the `alb.ingress.kubernetes.io/listen-ports` annotation to `[{"HTTP": 80}]` and create a CloudFront distribution using the ALB endpoint as the content origin. Using this approach, you can access the {product-short} using the CloudFront domain name. For more information about creating a CloudFront distribution, see https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/distribution-web-creating.html[Steps for creating a distribution].
+====
+--
+
+. (Optional) If an ALB or another ingress controller is used to expose your {product-short} instance and you are unaware of the DNS hostname, then identify the provisioned DNS name and update the {product-short} configuration using the following steps:
+
+.. Run the following command to find the ingress hostname:
++
+--
+[source]
+----
+$ kubectl get ingress rhdh-developer-hub   
+
+NAME                    CLASS    HOSTS                                                                 ADDRESS                                                               PORTS   AGE
+rhdh-developer-hub   <none>   k8s-myns-rhdhde-3dec682266-491020386.eu-north-1.elb.amazonaws.com   k8s-myns-rhdhde-3dec682266-491020386.eu-north-1.elb.amazonaws.com   80      51m
+----
+--
+
+.. Create a ConfigMap named `app-config-rhdh` using the following template:
++
+--
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: app-config-rhdh
+data:
+  "app-config-rhdh.yaml": |
+    app:
+      title: Red Hat Developer Hub
+      baseUrl: https://<rhdh_dns_name>
+    backend:
+      auth:
+        keys:
+          - secret: "${BACKEND_SECRET}"
+      baseUrl: https://<rhdh_dns_name>
+      cors:
+        origin: https://<rhdh_dns_name>
+----
+
+In the previous template, replace `<rhdh_dns_name>` with the value of {product-short} DNS name, such as `k8s-rhdhoper-myrhdh-f9ec8d3481-1192320380.eu-north-1.elb.amazonaws.com` or with the value of CloudFront DNS name, such as `d376s7j9emms3n.cloudfront.net` if CloudFront is used.
+--
+
+.. Create a Secret named `secrets-rhdh` and add a `BACKEND_SECRET` key with a `Base64-encoded` string as value:
++
+--
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: secrets-rhdh
+stringData:
+  # TODO: See https://backstage.io/docs/auth/service-to-service-auth/#setup:
+  # node -p 'require("crypto").randomBytes(24).toString("base64")'
+  BACKEND_SECRET: "R2FxRVNrcmwzYzhhN3l0V1VRcnQ3L1pLT09WaVhDNUEK"
+----
+Ensure that you use a unique name for each {product-short} instance.
+
+You can also use the following command to generate a key from the terminal:
+
+`node-p'require("crypto").randomBytes(24).toString("base64")'`
+--
+
+.. Update your custom resource created previously:
```

Comment:
Custom Resource will be created with the ConfigMap and Secret in the first steps. As such, it won't need to be updated again.

Reply:
Done